src/maestro/file_logger.py

```diff
@@ -1,10 +1,11 @@
 import uuid
 from datetime import datetime, UTC
 from pathlib import Path
+import os
 
 home_path = Path.home()
-if home_path == Path("/"):
-    DEFAULT_LOG_DIR = Path("/app/logs")
+if not os.access(home_path, os.W_OK):
+    DEFAULT_LOG_DIR = Path("/tmp/maestro/logs")
```

Comment:
these will get deleted if the user shuts down their laptop... maybe somewhere like `./logs` for persistence?